server/server.js

```diff
@@ -9,19 +9,6 @@ var axios = require('axios')
 var _ = require('lodash');
 _.mixin(require('congruence'));
```

Comment:
Remove unnecessary imports, and remove the dependencies that are no longer needed.